.github/workflows/CI.yml

```diff
@@ -0,0 +1,49 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  CI:
+    strategy:
+      matrix:
+        backend: ["OPENMP"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install deps
+        run: sudo apt-get install gfortran openmpi-bin libopenmpi-dev
+      - name: Checkout code
+        uses: actions/checkout@v2.2.0
+      - name: Checkout kokkos
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: kokkos/kokkos
+          ref: 3.1.00
+          path: kokkos
+      - name: Checkout Cabana
+        uses: actions/checkout@v2.2.0
+        with:
+          repository: ECP-copa/Cabana
+          ref: master
+          path: Cabana
+      - name: Build kokkos
+        working-directory: kokkos
+        run: |
+          cmake -B build -DCMAKE_INSTALL_PREFIX=$HOME/kokkos -DKokkos_CXX_STANDARD=14 -DKokkos_ENABLE_${{ matrix.backend }}=ON
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Build Cabana
+        working-directory: Cabana
+        run: |
+          cmake -B build -DCMAKE_INSTALL_PREFIX=$HOME/Cabana -DCMAKE_PREFIX_PATH="$HOME/kokkos" -DCabana_REQUIRE_${{ matrix.backend }}=ON
+          cmake --build build --parallel 2
+          cmake --install build
+      - name: Build
+        run: |
+          cmake -B build -DCMAKE_PREFIX_PATH="$HOME/kokkos;$HOME/Cabana"
+          cmake --build build --parallel 2
+          cd build && ctest --output-on-failure
```

Comment:
There are no unit tests, but the tutorial is set up to use gtest to make sure it least runs. Seems worth keeping

Reply:
Huh? `git grep -i gtest` returns nothing, where do we depend on gtest?

Reply:
You're right - it's missing in the top level CMakeLists, but all the tutorials already `add_test`. I should have included it in the previous PR